tests/analyze_assoc.R

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/Rscript
```

Comment:
Please use the same shebang line as the other executable R scripts:

!#/usr/bin/env Rscript

This finds the Rscript binary in your PATH, which is useful for say a custom built R.